png_navigation/NavigationEnvMsg request_env

---

bool is_set